http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20231116_add_auto_delete_threshold_to_app_table.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="qi" id="yonghao_20231116_add_auto_delete_threshold_to_app_table">
+        <addColumn tableName="APP">
+            <column name="autoDeleteThresholdInMinutes" defaultValue="0" type="SMALLINT">
+                <constraints nullable="true"/>
```

Comment:
think we should make it non-nullable..that way, rows without value will be populated with default value I think